tools/bug-report/pkg/config/config.go

```diff
@@ -50,28 +50,29 @@ const (
 //
 // Examples:
 //
-// 1. All pods in test-namespace with label "test=foo" but without label "private" (with any value):
-//      include:
-//        test-namespace/*/*/test=foo
-//      exclude:
-//        test-namespace/*/*/private
+//  1. All pods in test-namespace with label "test=foo" but without label "private" (with any value):
+//     include:
+//     test-namespace/*/*/test=foo
+//     exclude:
+//     test-namespace/*/*/private
 //
 // 2. Pods in all namespaces except "kube-system" with annotation "revision"
 // matching wildcard 1.6*:
-//      exclude:
-//        kube-system/*/*/*/revision=1.6*
+//
+//	exclude:
+//	  kube-system/*/*/*/revision=1.6*
 //
 // 3. Pods with "prometheus" in the name, except those with
 // the annotation "internal=true":
-//      include:
-//        */*/*prometheus*
-//      exclude:
-//        */*/*prometheus*/*/internal=true
 //
-// 4. Container logs for all containers called "istio-proxy":
-//      include:
-//        */*/*/*/*/istio-proxy
+//		include:
+//		  */*/*prometheus*
+//		exclude:
+//		  */*/*prometheus*/*/internal=true
 //
+//	 4. Container logs for all containers called "istio-proxy":
```

Comment:
I notice 4. is indented. Doesn't seem correct.

Reply:
yeah it renders wrong